lib/code_corps/onboarding/user_transition.ex

```diff
@@ -0,0 +1,10 @@
+defmodule CodeCorps.Onboarding.UserTransition do
```

Comment:
Wonder if `CodeCorps.Transition.UserState` is a better name?

Then we can also have a `Transition.PostStatus` or `PostStatus`.